### PR TITLE
MCC-189006 Create ConfigurationInfoHandler and UTs

### PR DESCRIPTION
**Pull request by chenghuang-mdsol**
@art2003 @echen-mdsol please review

### CODE REVIEW

File: Mct.RaveCommon/ConfigurationInfoHandler.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+				using (SqlCommand cmd = new SqlCommand("select Id,Tag,ConfigValue from [dbo].[Configuration]", sqlConnection))
+				{
+					return cmd.ExecuteReader();
+				}
+			}
+		}
+
+		internal virtual List<object> GetConfigurationInfoFromDb(IDataReader reader)
+		{
+			var result = new List<object>();
```
Comment (art2003) [inline Mct.RaveCommon/ConfigurationInfoHandler.cs]: should we check if reader is not null

File: Mct.RaveCommon/ConfigurationInfoHandler.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			var json = JsonConvert.SerializeObject(target, new StringEnumConverter());
+			expando = JsonConvert.DeserializeObject<ExpandoObject>(json, new ExpandoObjectConverter());
+			return expando;
+		}
+
+		[ExcludeFromCodeCoverage]
+		internal virtual IDataReader GetDataReaderByConnection(IDbConnection connection)
+		{
+			using (SqlConnection sqlConnection = (SqlConnection)connection)
+			{
```
Comment (art2003) [inline Mct.RaveCommon/ConfigurationInfoHandler.cs]: Should we pass connection string and open connection inside using statement
using (SqlConnection con = new SqlConnection(connectionString))

File: Mct.RaveCommon/ConfigurationInfoHandler.cs (hunk 1 above)
Comment (chenghuang-mdsol) [inline Mct.RaveCommon/ConfigurationInfoHandler.cs]: will do

File: Mct.RaveCommon/ConfigurationInfoHandler.cs (hunk 2 above)
Comment (chenghuang-mdsol) [inline Mct.RaveCommon/ConfigurationInfoHandler.cs]: will do

**Comment by chenghuang-mdsol** [comment]
@art2003 please check again. fixed the things you mentioned

**Comment by art2003** [comment]
LGTM, all comments addressed

**Pull request merged**